ui/tests/acceptance/task-fs-test.js

```diff
@@ -256,12 +296,40 @@ module('Acceptance | task fs', function(hooks) {
   });
 
   test('viewing a file', async function(assert) {
+    const node = server.db.nodes.find(allocation.nodeId);
+
+    server.logging = true;
```

Comment:
Is this meant to be left here?

Reply:
Hahaha, this is extra funny considering I'm removing it elsewhere in the same PR.